[2025-09-09 15:47:54,149] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:01:16,965] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:03:22,643] INFO django.utils.autoreload:265 — /app/backend/settings.py changed, reloading.
[2025-09-09 16:09:50,807] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:11:41,189] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:12:09,185] INFO django.utils.autoreload:265 — /app/shop/views.py changed, reloading.
[2025-09-09 16:12:10,529] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:12:50,427] INFO django.utils.autoreload:265 — /app/users/urls.py changed, reloading.
[2025-09-09 16:12:51,290] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:14:02,454] INFO django.utils.autoreload:265 — /app/orders/urls.py changed, reloading.
[2025-09-09 16:14:03,370] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:14:21,923] INFO django.utils.autoreload:265 — /app/orders/urls.py changed, reloading.
[2025-09-09 16:14:22,522] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:15:03,086] INFO django.utils.autoreload:265 — /app/payments/urls.py changed, reloading.
[2025-09-09 16:15:04,373] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:35:22,794] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:37:01,654] INFO django.utils.autoreload:265 — /app/payments/urls.py changed, reloading.
[2025-09-09 16:37:02,646] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:37:04,603] INFO django.utils.autoreload:265 — /app/payments/urls.py changed, reloading.
[2025-09-09 16:37:05,339] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:37:15,674] INFO django.utils.autoreload:265 — /app/payments/urls.py changed, reloading.
[2025-09-09 16:37:16,186] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:40:52,362] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:41:55,619] INFO django.utils.autoreload:265 — /app/backend/settings.py changed, reloading.
[2025-09-09 16:41:56,743] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:42:44,481] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:42:59,204] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:44:48,488] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:45:06,219] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:47:51,486] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-09 16:48:13,429] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:08:00,120] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:15:27,435] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:16:14,804] INFO django.utils.autoreload:265 — /app/shop/urls.py changed, reloading.
[2025-09-11 15:16:15,919] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:16:22,206] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:24:55,539] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:25:06,175] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:17,154] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:17,977] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:21,266] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:21,750] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:23,482] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:24,212] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:27,273] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:27,735] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:28,970] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:29,525] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:32,164] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:32,678] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:33,922] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:34,651] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:36,241] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:36,846] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 15:43:40,644] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 15:43:41,587] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:07:52,745] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:07:53,600] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:08:01,103] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:08:01,601] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:08:04,124] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:08:04,752] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:08:08,059] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:08:08,646] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:08:18,682] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:08:19,261] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:08:35,829] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:08:36,550] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:09:09,510] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:09:10,246] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:09:47,596] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:09:48,167] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:12:07,008] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:12:07,782] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:14:53,821] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:14:54,498] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:15:38,123] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:15:38,870] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:15:51,495] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:15:52,143] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:16:16,642] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-11 16:16:17,364] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:17:24,417] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:17:25,300] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:18:35,603] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:18:36,267] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:19:17,698] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:19:18,258] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:19:39,097] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:19:39,649] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:20:02,102] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:20:02,765] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:20:14,845] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:20:15,625] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:21:25,548] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:21:26,290] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:22:51,044] INFO django.utils.autoreload:265 — /app/users/views.py changed, reloading.
[2025-09-11 16:22:51,613] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:30:59,923] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:32:10,599] INFO django.utils.autoreload:265 — /app/products/admin.py changed, reloading.
[2025-09-11 16:32:11,029] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-11 16:33:11,929] INFO django.utils.autoreload:265 — /app/products/admin.py changed, reloading.
[2025-09-11 16:33:12,363] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-13 12:42:10,787] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-13 12:42:49,636] INFO django.utils.autoreload:265 — /app/orders/admin.py changed, reloading.
[2025-09-13 12:42:50,030] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-13 12:42:57,633] INFO django.utils.autoreload:265 — /app/products/admin.py changed, reloading.
[2025-09-13 12:42:58,037] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-13 12:45:45,885] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-13 12:48:45,049] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-15 15:00:11,541] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
<<<<<<< Updated upstream
=======
[2025-09-16 15:28:12,354] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 15:28:30,426] INFO django.utils.autoreload:265 — /app/products/urls.py changed, reloading.
[2025-09-16 15:28:31,889] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 15:30:50,437] INFO django.utils.autoreload:265 — /app/cart/admin.py changed, reloading.
[2025-09-16 15:30:51,148] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
<<<<<<< Updated upstream
<<<<<<< Updated upstream
<<<<<<< Updated upstream
<<<<<<< Updated upstream
[2025-09-16 15:59:05,592] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
>>>>>>> Stashed changes
[2025-09-16 16:02:08,972] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
=======
[2025-09-16 16:02:43,946] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
>>>>>>> Stashed changes
[2025-09-16 16:04:53,147] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
=======
[2025-09-16 16:05:51,679] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
>>>>>>> Stashed changes
[2025-09-16 16:06:10,408] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
=======
[2025-09-16 16:09:45,080] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
>>>>>>> Stashed changes
[2025-09-16 16:10:36,266] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:21:06,084] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
=======
[2025-09-16 16:18:51,232] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:21:46,495] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
>>>>>>> Stashed changes
[2025-09-16 16:22:36,108] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:26:09,646] INFO django.utils.autoreload:265 — /app/products/urls.py changed, reloading.
[2025-09-16 16:26:14,260] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:26:37,427] INFO django.utils.autoreload:265 — /app/cart/admin.py changed, reloading.
[2025-09-16 16:26:38,325] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:26:40,484] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:26:47,531] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:26:50,961] WARNING django.request:253 — Not Found: /
[2025-09-16 16:26:50,965] WARNING django.server:213 — "GET / HTTP/1.1" 404 3056
[2025-09-16 16:27:01,843] INFO django.server:213 — "GET /admin/ HTTP/1.1" 302 0
[2025-09-16 16:27:01,902] INFO django.server:213 — "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4173
[2025-09-16 16:27:09,652] ERROR django.request:253 — Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedColumn: column users_myuser.phone_number does not exist
LINE 1: ...yuser"."date_joined", "users_myuser"."is_seller", "users_myu...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/contrib/admin/sites.py", line 449, in login
    return LoginView.as_view(**defaults)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/decorators/debug.py", line 143, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/contrib/auth/views.py", line 89, in dispatch
    return super().dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/generic/edit.py", line 150, in post
    if form.is_valid():
       ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/forms/forms.py", line 206, in is_valid
    return self.is_bound and not self.errors
                                 ^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/forms/forms.py", line 201, in errors
    self.full_clean()
  File "/usr/local/lib/python3.12/site-packages/django/forms/forms.py", line 338, in full_clean
    self._clean_form()
  File "/usr/local/lib/python3.12/site-packages/django/forms/forms.py", line 354, in _clean_form
    cleaned_data = self.clean()
                   ^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/decorators/debug.py", line 75, in sensitive_variables_wrapper
    return func(*func_args, **func_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/contrib/auth/forms.py", line 366, in clean
    self.user_cache = authenticate(
                      ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/views/decorators/debug.py", line 75, in sensitive_variables_wrapper
    return func(*func_args, **func_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/contrib/auth/__init__.py", line 114, in authenticate
    user = backend.authenticate(request, **credentials)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/contrib/auth/backends.py", line 65, in authenticate
    user = UserModel._default_manager.get_by_natural_key(username)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/contrib/auth/base_user.py", line 37, in get_by_natural_key
    return self.get(**{self.model.USERNAME_FIELD: username})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/models/query.py", line 629, in get
    num = len(clone)
          ^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/models/query.py", line 366, in __len__
    self._fetch_all()
  File "/usr/local/lib/python3.12/site-packages/django/db/models/query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.12/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.12/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: column users_myuser.phone_number does not exist
LINE 1: ...yuser"."date_joined", "users_myuser"."is_seller", "users_myu...
                                                             ^

[2025-09-16 16:27:09,658] ERROR django.server:213 — "POST /admin/login/?next=/admin/ HTTP/1.1" 500 239905
[2025-09-16 16:29:00,778] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:29:45,942] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:31:02,702] WARNING django.request:253 — Not Found: /
[2025-09-16 16:31:02,706] WARNING django.server:213 — "GET / HTTP/1.1" 404 3056
[2025-09-16 16:31:06,541] INFO django.server:213 — "GET /admin/ HTTP/1.1" 302 0
[2025-09-16 16:31:06,624] INFO django.server:213 — "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4173
[2025-09-16 16:31:09,986] INFO django.server:213 — "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4342
[2025-09-16 16:31:14,208] INFO django.server:213 — "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4339
[2025-09-16 16:33:35,369] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-16 16:33:36,011] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:33:38,553] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:34:07,213] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-16 16:34:08,278] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:34:10,984] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:34:57,425] INFO django.utils.autoreload:265 — /app/users/models.py changed, reloading.
[2025-09-16 16:34:58,266] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:35:01,494] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 16:35:25,471] INFO django.server:213 — "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
[2025-09-16 16:35:25,541] INFO django.server:213 — "GET /admin/ HTTP/1.1" 200 11611
[2025-09-16 16:35:25,599] INFO django.server:213 — "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
[2025-09-16 16:35:25,724] INFO django.server:213 — "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[2025-09-16 16:35:25,724] INFO django.server:213 — "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[2025-09-16 17:10:39,778] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
[2025-09-16 17:10:41,550] WARNING django.request:253 — Not Found: /
[2025-09-16 17:10:41,555] WARNING django.server:213 — "GET / HTTP/1.1" 404 3056
[2025-09-16 17:10:42,520] WARNING django.request:253 — Not Found: /favicon.ico
[2025-09-16 17:10:42,521] WARNING django.server:213 — "GET /favicon.ico HTTP/1.1" 404 3107
[2025-09-16 17:10:47,407] INFO django.server:213 — "GET /admin/ HTTP/1.1" 200 11611
[2025-09-16 17:10:54,176] INFO django.server:213 — "GET /admin/products/product/add/ HTTP/1.1" 200 22036
[2025-09-16 17:10:54,235] INFO django.server:213 — "GET /static/admin/css/forms.css HTTP/1.1" 200 8525
[2025-09-16 17:10:54,271] INFO django.server:213 — "GET /static/admin/js/actions.js HTTP/1.1" 200 8076
[2025-09-16 17:10:54,299] INFO django.server:213 — "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
[2025-09-16 17:10:54,318] INFO django.server:213 — "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
[2025-09-16 17:10:54,324] INFO django.server:213 — "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
[2025-09-16 17:10:54,330] INFO django.server:213 — "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9777
[2025-09-16 17:10:54,332] INFO django.server:213 — "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
[2025-09-16 17:10:54,332] INFO django.server:213 — "GET /static/admin/js/core.js HTTP/1.1" 200 6208
[2025-09-16 17:10:54,342] INFO django.server:213 — "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2025-09-16 17:10:54,343] INFO django.server:213 — "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314
[2025-09-16 17:10:54,363] INFO django.server:213 — "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171
[2025-09-16 17:10:54,488] INFO django.server:213 — "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
[2025-09-16 17:10:54,510] INFO django.server:213 — "GET /static/admin/css/widgets.css HTTP/1.1" 200 11991
[2025-09-16 17:10:54,526] INFO django.server:213 — "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
[2025-09-16 17:12:30,951] INFO django.server:213 — "POST /admin/products/product/add/ HTTP/1.1" 200 22618
[2025-09-16 17:12:31,015] INFO django.server:213 — "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2025-09-16 17:12:47,222] INFO django.server:213 — "GET /admin/products/subcategory/add/?_to_field=id&_popup=1 HTTP/1.1" 200 6106
[2025-09-16 17:12:47,304] INFO django.server:213 — "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2025-09-16 17:12:50,281] INFO django.server:213 — "GET /admin/products/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 4389
[2025-09-16 17:12:50,329] INFO django.server:213 — "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2025-09-16 17:12:52,927] INFO django.server:213 — "POST /admin/products/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 323
[2025-09-16 17:12:52,956] INFO django.server:213 — "GET /static/admin/js/popup_response.js HTTP/1.1" 200 532
[2025-09-16 17:12:55,932] INFO django.server:213 — "POST /admin/products/subcategory/add/?_to_field=id&_popup=1 HTTP/1.1" 200 324
[2025-09-16 17:12:59,030] INFO django.server:213 — "GET /admin/products/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 4389
[2025-09-16 17:12:59,078] INFO django.server:213 — "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2025-09-16 17:13:02,323] INFO django.server:213 — "POST /admin/products/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 323
[2025-09-16 17:13:06,965] INFO django.server:213 — "POST /admin/products/product/add/ HTTP/1.1" 302 0
[2025-09-16 17:13:07,029] INFO django.server:213 — "GET /admin/products/product/1/change/ HTTP/1.1" 200 22754
[2025-09-16 17:13:07,134] INFO django.server:213 — "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[2025-09-16 17:13:07,273] INFO django.server:213 — "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
[2025-09-16 17:13:12,960] WARNING django.request:253 — Not Found: /
[2025-09-16 17:13:12,961] WARNING django.server:213 — "GET / HTTP/1.1" 404 3056
[2025-09-16 17:13:15,993] INFO django.server:213 — "GET /admin/ HTTP/1.1" 200 13039
[2025-09-16 17:13:17,542] INFO django.server:213 — "GET /shop HTTP/1.1" 301 0
[2025-09-16 17:13:17,616] ERROR django.request:253 — Internal Server Error: /shop/
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/core/handlers/base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/django/template/response.py", line 114, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rest_framework/response.py", line 74, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rest_framework/renderers.py", line 731, in render
    context = self.get_context(data, accepted_media_type, renderer_context)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rest_framework/renderers.py", line 687, in get_context
    'content': self.get_content(renderer, data, accepted_media_type, renderer_context),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rest_framework/renderers.py", line 417, in get_content
    content = renderer.render(data, accepted_media_type, renderer_context)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rest_framework/renderers.py", line 100, in render
    ret = json.dumps(
          ^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rest_framework/utils/json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "/usr/local/lib/python3.12/json/encoder.py", line 202, in encode
    chunks = list(chunks)
             ^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/json/encoder.py", line 432, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)
  File "/usr/local/lib/python3.12/json/encoder.py", line 406, in _iterencode_dict
    yield from chunks
  File "/usr/local/lib/python3.12/json/encoder.py", line 440, in _iterencode
    yield from _iterencode(o, _current_indent_level)
  File "/usr/local/lib/python3.12/json/encoder.py", line 430, in _iterencode
    yield from _iterencode_list(o, _current_indent_level)
  File "/usr/local/lib/python3.12/json/encoder.py", line 326, in _iterencode_list
    yield from chunks
  File "/usr/local/lib/python3.12/json/encoder.py", line 439, in _iterencode
    o = _default(o)
        ^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/rest_framework/utils/encoders.py", line 67, in default
    return super().default(obj)
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/json/encoder.py", line 180, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type Product is not JSON serializable
[2025-09-16 17:13:17,620] ERROR django.server:213 — "GET /shop/ HTTP/1.1" 500 161692
[2025-09-16 17:13:39,049] INFO django.server:213 — "GET /admin/ HTTP/1.1" 200 13039
[2025-09-16 17:14:00,061] INFO django.utils.autoreload:667 — Watching for file changes with StatReloader
